cli/tests/unit/os_test.ts

```diff
@@ -42,13 +42,11 @@ unitTest({ perms: { env: true } }, function avoidEmptyNamedEnv(): void {
   assertThrows(() => Deno.env.delete("a\0a"), TypeError);
 });
 
-unitTest(function envPermissionDenied1(): void {
-  assertThrows(() => {
-    Deno.env.toObject();
-  }, Deno.errors.PermissionDenied);
+unitTest(function envToObjectWithoutPermission(): void {
+  assertEquals(Deno.env.toObject(), {});
```

Comment:
This change seems strange to me, why doesn't it throw anymore?

Reply:
this is related to what @kt3k reviewed above. i will fix it later today